vm/src/stdlib/symtable.rs

```diff
@@ -28,7 +27,9 @@ fn symtable_symtable(
     mode: PyStringRef,
     vm: &VirtualMachine,
 ) -> PyResult<PySymbolTableRef> {
-    let mode = get_compile_mode(vm, &mode.value)?;
+    let mode = mode.as_str().parse().map_err(|_| {
+        vm.new_value_error("symtable() arg 3 must be 'exec' or 'eval' or 'single'".to_owned())
```

Comment:
I think it is fine to just display the inner error here, and slightly deviate from the exact CPython error message.